packages/components/src/utils/colors-values.js

```diff
@@ -36,34 +36,57 @@ const ADMIN = {
 		'var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #2145e6))',
 };
 
-const UI = {
-	background: white,
-	backgroundDisabled: GRAY[ 100 ],
-	border: GRAY[ 600 ],
-	borderHover: GRAY[ 700 ],
-	borderFocus: ADMIN.theme,
-	borderDisabled: GRAY[ 400 ],
-	textDisabled: GRAY[ 600 ],
-	textDark: white,
-
-	// Matches @wordpress/base-styles
-	darkGrayPlaceholder: rgba( GRAY[ 900 ], 0.62 ),
-	lightGrayPlaceholder: rgba( white, 0.65 ),
-};
-
+// Should match packages/components/src/utils/theme-variables.scss
 const THEME = {
 	accent: ADMIN.theme,
 	accentDarker10: ADMIN.themeDark10,
+	background: `var(--wp-components-color-background, ${ white })`,
+	foreground: `var(--wp-components-color-foreground, ${ GRAY[ 900 ] })`,
+	gray: {
+		900: `var(--wp-components-color-gray-900, ${ GRAY[ 900 ] })`,
```

Comment:
It doesn't look like `--wp-components-color-gray-900` is defined in the `Theme` component. I can't remember if there a specific reason behind that decision, or if we just forgot to add it.

Reply:
Good catch! The reason was that `gray-900` is the same as `foreground` and thus redundant. Addressed in 8b321ab by keeping a deprecated alias that nudges people to use `foreground`.

Ultimately I feel like the scale should be named like `foreground-600` etc. instead of `gray-600`, but I'm not quite ready to commit until we do a bit more actual migrations and exploration.